src/k8s/pkg/k8sd/types/cluster_config_validate.go

```diff
@@ -20,6 +21,24 @@ func validateCIDRs(cidrString string) error {
 	return nil
 }
 
+// isValidAddress checks if a given address is valid. Allows with and without schema.
+func isValidAddress(address string) bool {
```

Comment:
This looks like a function that could be used in many places. Have you considered putting this in a common place like utils?

Reply:
Let's keep it here for now, it is kind of specific for the validator (allows schemaless etc.).